api/src/main/java/org/openmrs/module/pihcore/PihCoreActivator.java

```diff
@@ -220,6 +222,48 @@ public void setGlobalProperties(Config config) {
         }
     }
 
+    public User setupCommCareUser(){
+
+	    User user = null;
+        String commCareUserName = Context.getRuntimeProperties().getProperty("commcare.username", "commCareUser");
+        String commCareUserPassword = Context.getRuntimeProperties().getProperty("commcare.password", "CommCare12");
```

Comment:
We should make the default values here blank and not install a user if the values aren't specified in the runtime properties... otherwise we could inadvertently create a user with the username/password pair in plaintext in a public repo... :)  Then we'd only set these runtime properties in Puppet for the servers that need this user.